core-java/src/main/java/com/github/wasiqb/boyka/builders/ApiResponse.java

```diff
@@ -98,6 +101,23 @@ public BooleanSubject verifyBooleanField (final String expression) {
         return assertThat (getResponseData (expression, Boolean.class));
     }
 
+    /**
+     * Verify header in response.
+     *
+     * @param key header key
+     *
+     * @return {@link StringSubject} instance
+     */
+    public StringSubject verifyHeader (final String key) {
+        LOGGER.traceEntry ();
+        LOGGER.info ("Verifying response Header: {}", key);
+        LOGGER.traceExit ();
+        if (getHeaders ().get (key) == null) {
```

Comment:
Please use `containsKey` method and not `get` in this `if` condition.

Reply:
Feedback incorporated